### DISCUSSION TITLE
Q-Table with hiding First & Last pagination buttons but displaying number of records

### DISCUSSION THREAD

**Discussion by L-S-ONG** [quasarframework/quasar#13676] [General - Components / Directives / etc] [answered]
Hi Quasarians,

I have been trying to work on hiding just the **First & Last** page buttons while displaying only the **Next & Previous** page buttons + the **number of records**

![image](https://user-images.githubusercontent.com/56607625/173275712-65a637e7-3f62-4025-b533-19998066849c.png)

I have tried using Pagination slot and managed to get rid of the **First & Last** Page button but by doing so, the **numbers of records** went missing as well

![image](https://user-images.githubusercontent.com/56607625/173275985-c4148353-b8b0-48b0-9d8e-57bfb1de3c33.png)

Any idea how am I able to display just the **numbers of records** along with **Next & Previous** buttons?

**Comment by pratik227** [comment, marked as answer]
Use Pagination Slot.

Doc - https://quasar.dev/vue-components/table#pagination-slot

https://codepen.io/Pratik__007/pen/gOvZmQX

**Reply by L-S-ONG** [reply]
Hi Pratik, 

I have tried pagination slot, but as mentioned in my post, I have tried using pagination slot and the numbers of records went missing, how can I retain the number of records in this case?

**Reply by L-S-ONG** [reply]
To be more specific, this is what I want to achieve

![image](https://user-images.githubusercontent.com/56607625/173293793-2d54d36d-2d65-4ad0-9da6-8f3d82a17e85.png)

with Pagination slot, it would only look like this 

![image](https://user-images.githubusercontent.com/56607625/173293839-84cbb950-93a3-4c0e-b1a6-c6b0268e5b6d.png)

**Reply by pratik227** [reply]
Updated Codepen Please Check

You need to write custom logic for that based on rowPerPage and PagesNumbers etc.

**Reply by L-S-ONG** [reply]
Thanks a lot, I was able to get the numbers of records based on the example provided in your CodePen.

I have modified a little and it works

`{{ ((scope.pagination.page - 1 )* scope.pagination.rowsPerPage)+1}} - {{scope.pagination.page*scope.pagination.rowsPerPage>rows.length?rows.length+1:scope.pagination.page*scope.pagination.rowsPerPage}} of {{scope.pagesNumber}}`

this was what I edited (minor)

`{{ ((scope.pagination.page - 1 )* scope.pagination.rowsPerPage)+1}} - {{scope.pagination.page*scope.pagination.rowsPerPage>rows.length?rows.length:scope.pagination.page*scope.pagination.rowsPerPage}} of {{rows.length}}`
